### PR TITLE
BaSP-TG-01: Super Admin table - Resources List

### PR DESCRIPTION
**Pull request by Juancapp**
**Show a table with the list of resources of the assigned entity, saved in the database.**

**For each row, you must be able to:**

- Edit an item.
- Delete an item.
- Show success and error messages.

 **For the whole table:**
- Create a redirect button to 'Add feature'

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/jest-dom": "^5.15.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "babel-plugin-macros": "^3.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.1",
     "web-vitals": "^1.1.2"
```
Comment (leimeter-joaquin) [inline package.json:17]: Remove every change to package.json

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  Name: ${name}
+  Last Name: ${lastName}
+  Email: ${email}
+  Password: ${password}
+  Dni: ${dni}
+  Phone: ${phone}
+  `;
+  };
+
+  if (idRegEx.test(product)) {
```
Comment (leimeter-joaquin) [inline src/Components/SuperAdmins/Form/index.js:33]: Place the if inside the useEffect

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+          name: nameValue,
+          lastName: lastNameValue,
+          email: emailValue,
+          password: passwordValue,
+          dni: dniValue,
+          phone: phoneValue
+        })
+      });
+      const data = await response.json();
+      console.log(data);
```
Comment (TomasBettini) [inline src/Components/SuperAdmins/Form/index.js:65]: You must delete this console.log

**Review by TomasBettini — DISMISSED** [review]
Nice code!

**Review by juliancit0 — CHANGES_REQUESTED** [review]

**Review by juliancit0 — DISMISSED** [review]

File: src/Components/SuperAdmins/Form/index.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+            <input
+              id="input-phone"
+              name="phone"
+              required
+              value={phoneValue}
+              onChange={changePhone}
+            />
+          </div>
+          <div>
+            <a href={'http://localhost:3000/super-admins'}>
```
Comment (leimeter-joaquin) [inline src/Components/SuperAdmins/Form/index.js:206]: use the env variable, not localhost

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]
Fix that, everything else seems great!

**Review by leimeter-joaquin — DISMISSED** [review]

**Review by Alfondalix — DISMISSED** [review]

**Review by leimeter-joaquin — APPROVED** [review]

**Review by amilcarHernandez24 — APPROVED** [review]
Good job.

**Pull request merged**